src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const price = PRICE_PER_DAY * days;
```

Comment:
A previous reviewer made a good comment, but you don't have to uppercase the variable.
Uppercase is used only for static constants, that is, which are not calculated and do not change.
```js
const totalPrice = PRICE_PER_DAY * days;
```